asset-tracking-location-replaying/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "asset-tracking-location-replaying",
+  "version": "1.0.0",
+  "description": "CLI tool that replays location updates",
+  "main": "index.ts",
+  "scripts": {
+    "start": "./node_modules/ts-node/dist/bin.js src/index.ts"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "ably": "^1.2.4",
+    "commander": "^7.0.0",
+    "death": "^1.1.0",
+    "ts-node": "^10.0.0",
+    "typescript": "^4.3.2"
```

Comment:
I'd definitely recommend adding a tsconfig.json. Even just by running `npx tsc --init`. I think some developer's toolchains might not work without one and it's better to be explicit about what the rules are.

Reply:
https://github.com/ably/ably-asset-tracking-common/pull/25/commits/ab37bc49617bdc8a27968972e24eb3ab506b2b16